### ISSUE TITLE
Images sub-fields in a complex field are not displayed, but value is saved

### ISSUE THREAD

**Issue by psaikali** [htmlburger/carbon-fields#973]
Hey there!

I'm using CF `"htmlburger/carbon-fields": "^3.2"` with WP 5.7

I have a complex field with an `image` sub-field in it, registered as such:
```php
Field::make( 'complex', 'profile_tags', __( 'Titre', 'freebulle' ) )
	->add_fields( [
		Field::make( 'checkbox', 'is_active', __( 'Actif ?', 'freebulle' ) )
			->set_option_value( 'yes' )
			->set_width( 12 ),
		Field::make( 'select', 'group', __( 'Groupe', 'freebulle' ) )
			->set_required( true )
			->set_options( Config::get_profile_tags_groups() )
			->set_width( 22 ),
		Field::make( 'text', 'slug', __( 'Identifiant unique', 'freebulle' ) )
			->set_required( true )
			->set_width( 22 ),
		Field::make( 'text', 'name', __( 'Nom', 'freebulle' ) )
			->set_required( true )
			->set_width( 22 ),
		Field::make( 'image', 'icon', __( 'Icône', 'freebulle' ) )
			->set_required( true )
			->set_type( [ 'image' ] )
			->set_width( 22 ),
	] )
	->setup_labels( [
		'singular_name' => __( 'Tag', 'freebulle' ),
		'plural_name'   => __( 'Tags', 'freebulle' ),
	] )
	->set_header_template( 'Tag n°<%- $_index+1 %>' )
```

For the `icon` field, the image IDs are correctly saved in the database. But when the page is refreshed, the field appears empty.

You can see an example here: on the left side, `_profile_tags|icon|%|%|value` options **do** have a value, but the fields appear empty on the right side:
![Screen 2021-03-10 à 06 46 30](https://user-images.githubusercontent.com/1606655/110582364-6839c680-816c-11eb-99de-86116613f3e6.png)

**Comment by ProjetKotov** [comment]
I have the same issue

**Comment by jorostoyanov** [comment]
Hi Guys,

I've tested the same configuration, however, on my side the images are being shown.

Can you clarify which exact version of Carbon Fields are you using? Open vendor/htmlburger/carbon-fields/package.json and paste the version here.

Can you open DevTools and let me know if there are any errors?

Do you see requests being made to `https://{YOURSITE}/wp-json/carbon-fields/v1/attachment?type=id&value=..`

**Comment by psaikali** [comment]
Thanks @jorostoyanov,

I've fixed my issue, it was totally my fault 😆 
I'm using the `JWT Auth` plugin (https://github.com/usefulteam/jwt-auth) which blocks all Rest API requests if it does not have a valid JWT header in it.
I did not know that a CF media field send API requests to get the attachment data.
I've whitelisted the `/carbon-fields/v1/` requests using the `jwt_auth_default_whitelist` and it is now working fine.

Thank you!
